AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -50,6 +50,9 @@ disk.yandex.*##.mail360-banner
 !-----------------------------------------------
 !
 !
+!
+! https://github.com/AdguardTeam/AdguardFilters/issues/103031
+makuring.com##.shtb2.tbrsp
```

Comment:
You'll block their images. I'd use
```
makuring.com#?#.sponsored:has(> div.textwidget > div[class] > div.cell > a[href^="https://www.youtube.com/channel/"])
```
for the `Annoyances - subscriptions`, but I need a review by @Yuki2718.
Also add `makuring.com##.center > p.nospace` to the `AdGuard Japanese - specific`.

Reply:
ok.

Reply:
These are links to thier social media and are not redundant. We don't block them in principle, do we @zloyden ?

Reply:
Well, found redundant links at the bottom. Hmm...

Reply:
They have no recommendations, just social links? If so, we should remove the rule.

Reply:
We remove social links if they have:
1. Recommendations
2. Aggressive placement in the content and they also duplicate links on the header / footer.

Reply:
@SKEIDs Please remove `akuring.com#?#.sponsored:has(> div.textwidget > div[class] > div.cell > a[href^="https://www.youtube.com/channel/"])`.